ansible/playbooks/utils/check_volume.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Checking for necessary --extra_vars
+  hosts: all
+  vars:
+    example:
+      - "-e \"{'VOLUME_DEVICE':'/vol1', 'VOLUME_MKFS_DEVICE':True, 'VOLUME_DEVICE_TYPE': 'ext4'}\""
+
+  tasks:
+
+    - fail: msg="Must set VOLUME_DEVICE. Pass a value in via `-e` as JSON {{example[0]}}"
+      when: "{{ VOLUME_DEVICE is undefined }}"
+
+    - fail: msg="Must set VOLUME_DEVICE_TYPE. Pass a value in via `-e` as JSON {{example[0]}}"
+      when: "{{ VOLUME_DEVICE_TYPE is undefined }}"
+
+
+- name: Playbook to test volume before mounting
```

Comment:
- I think you mean "play"
- The role that you call actually ensures the volume is formatted with a filesystem, "test" implies it doesn't make changes

So, I would call this "Play to ensure filesystem is present before mounting".